codebuild/ci/release-ci.yml

```diff
@@ -27,7 +27,18 @@ phases:
   build:
     commands:
       - VERSION_HASH="$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)-$CODEBUILD_RESOLVED_SOURCE_VERSION"
-#      See https://docs.aws.amazon.com/codebuild/latest/userguide/build-env-ref-env-vars.html
+
+      # Remove any old artifacts with the same commit ID. This allows CI to run more than once for the same commit
+      - |
+        aws codeartifact delete-package-versions --domain $DOMAIN \
+          --repository $REPOSITORY \
+          --format maven \
+          --namespace $NAMESPACE \
+          --package $PACKAGE \
+          --versions $VERSION_HASH \
+          --region $REGION;
+
+      # See https://docs.aws.amazon.com/codebuild/latest/userguide/build-env-ref-env-vars.html
       - echo "Setting version in POM to $VERSION_HASH"
       - mvn versions:set -DnewVersion="$VERSION_HASH" --no-transfer-progress
       - echo "Version is now $(grep version pom.xml | head -n 1 | sed -n 's/[ \t]*<version>\(.*\)<\/version>/\1/p')"
```

Comment:
Can we update the output on L61,69 now? If Codebuild fails, it must be for some other reason now.

Reply:
Good catch, updated